README.md

```diff
@@ -147,7 +147,7 @@ make install DESTDIR=/path/to/dest
 ## Contact and discussion forums
 
 OSTree has a [mailing list](https://mail.gnome.org/archives/ostree-list/) and
-there is also an `#ostree` channel on FreeNode.  However, asynchronous+logged
+there is also an `#ostree` channel on irc.libera.chat.  However, asynchronous+logged
```

Comment:
Minor: I think the stylized name is "Libera.Chat".

Reply:
Pushed an updated version which turns this into an actual link (I hope I’ve got the link syntax right)